### ISSUE TITLE
N64 SM64 Heavily Modified Roms

### ISSUE THREAD

**Issue by DragonSoul2875** [KMFDManic/NESC-SNESC-Modifications#101]
### Step 1: Describe your setup:

  * hakchi2 CE version: v1.10
  * retroarch used (filename): _km_retroarch_171_xtreme_3_13_18
  * hmod(s) used by the game you are having problems with (filenames): _km_mupen64plus_2_27_18
  
### Step 2: Describe the problem:
  
  * Sorry if this is answered before however SM64 rom hacks such as Last Impact (which I know for sure throws a c7) and Star Road or really any rom that uses an 8MB memory size does not work or at least work well on retroarch. If the 8mb memory size is a feature already how do I access it? Otherwise would it be possibly to implement such a feature?

**Comment by KMFDManic** [comment]
Unfortunately, hacks, especially for N64 have proven to not be able to work well for the Mini.  Have you tried running them on PC?

**Comment by DragonSoul2875** [comment]
Yes, they work fine on any project 64 and mupen. As long as the settings for the rom are 8mb memory size.

**Comment by KMFDManic** [comment]
I will look into things today, and let you know the results.  Are there any "standard" releases that you are aware of that use 8mb?

**Comment by DragonSoul2875** [comment]
Not that I'm aware of. Most things don't need that much memory space. Just because of rom hack's textures and just sheer size make it necessary.

**Comment by KMFDManic** [comment]
This is where things get interesting.  Using Expansion Pack, technically gives the 4MB extra memory.  You shouldn't be able to even run Majora's Mask, unless Expansion Pack is being used.  Mupen64Plus is supposed to be using the Expansion Pack.  But, it appears to be bugged, and not properly using 8MB of memory!  Sort of puts a damper on things, having a broken option, if that is the case.  I have seen issues with the Expansion Pack option on PC, in the past, too!  So, basically, if you can load Majora's Mask with Mupen and/or Glupen, then, whichever 8MB Memory Required Games that WILL work, should work with that exact same Core:)
